### PR TITLE
libisns: remove sighold and sigrelse

### PR DESCRIPTION
**Pull request by neheb**
The man page says that these are deprecated. Use sugprocmask as a replacement.

### CODE REVIEW

File: include/libisns/util.h [hunk 1]
```diff
@@ -41,14 +41,20 @@ char *		print_size(unsigned long);
  */
 static inline void signals_hold(void)
 {
-	sighold(SIGTERM);
-	sighold(SIGINT);
+	sigset_t s;
+
```
Comment (gonzoleeman) [inline include/libisns/util.h:45]: Should you have a sigemptyset() here, to initialize your signal set to empty?

File: include/libisns/util.h [hunk 2]
```diff
@@ -41,14 +41,20 @@ char *		print_size(unsigned long);
...
+	sigaddset(&s, SIGINT);
+	sigprocmask(SIG_BLOCK, &s, 0);
 }
 
 static inline void signals_release(void)
 {
-	sigrelse(SIGTERM);
-	sigrelse(SIGINT);
+	sigset_t s;
+
```
Comment (gonzoleeman) [inline include/libisns/util.h:55]: ditto on sigemptyset() here.

**Review by gonzoleeman — CHANGES_REQUESTED** [review]

**Comment by neheb** [comment]
Done.

**Comment by gonzoleeman** [comment]
Thank you!

**Pull request merged**